packages/OmegaPrint-Tests.package/TestObjectArrayLiteral.class/instance/testWithSingleObjectArrayLiteralElementWithPoint.st

```diff
@@ -0,0 +1,6 @@
+base
+testWithSingleObjectArrayLiteralElementWithPoint
+
+	self
+		prettify: '{statement.}'
```

Comment:
Müssen wir da den Punkt testen?

Reply:
Der Punkt entscheidet, in welchen der beiden Faelle der Regel
```smalltalk
ObjectArrayLiteralElement =
	Statement "." --normal
	| Statement --finalExpression
```
wir gehen. Mit Punkt gehen wir garantiert in den oberen Fall. Wir wollen naemlich sicherstellen, dass, egal welchen Pfad wir waehlen, nach dem letzten `ObjectArrayLiteralElement` kein Punkt gesetzt wird.